src/containers/AccountManagement.jsx

```diff
@@ -53,10 +54,30 @@ export default class AccountManagement extends Component {
       })
   }
 
+  validInfos (input) {
+    const { t } = this.context
+    // Check if field is empty or not
+    if (input.target.value !== '') {
```

Comment:
@CPatchane, yes I know, but it does not change the fact that it should not be called here. I see it as a kind of `console.error`.